services/buildbot/master/txbuildbot/web.py

```diff
@@ -98,7 +97,7 @@ def body(self, req):
 
             builds = sorted([
                     build for build in builder.getCurrentBuilds()
-                    if build.getSourceStamp().branch in map_branches(branches)
+                    if build.getSourceStamps()[0].branch in map_branches(branches)
```

Comment:
True.

https://github.com/buildbot/buildbot/blob/cdfb4a2934be456c5ad2820feebc10dd08480677/master/buildbot/status/build.py#L86

will check what should be done if as the current code looks like it returns an empty dict